src/test/java/game/map/MapTest.java

```diff
@@ -17,6 +12,14 @@
 import java.util.Arrays;
 import java.util.List;
 
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.anyOf;
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
```

Comment:
It's a good practise to keep import static before all import. But it's fine, Joyjit said there is some library to format the code and it could help. You can leave it as-is (no need to make any changes).